Sitecore/jss
# [JSS] [NextJs] Sitemap query limits to 10 by default

## Bug Description
One of the late Edge schema changes included a change in pagination (switched to cursor-based), which also changed the default page size to 10 (previously, unlimited). We need to implement pagination in our sitemap service, and leave it up to the customer to override if needed. 

## How to test
Create Next.js site with more than 10 pages in Sitecore
Perform production build (`npm run build`)
Expected Result
Pre-generated output includes all pages.
